azure-pipelines.yml

```diff
@@ -112,7 +107,7 @@ trigger: none
 # verifying the build and deploying, and Linux OS for all other jobs for faster 
 # execution.
 pool:
-  vmImage: 'ubuntu-20.04'
+  vmImage: 'ubuntu-latest'
```

Comment:
```suggestion
  vmImage: 'node:10.14.2'
```

Reply:
@gunzip do you confirm `node:10.14.2`?

Reply:
it looks like there's no such hosted agent on azure devops https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml but we need to build on node 10.x to avoid installing packages that have node 12.x requirements

Reply:
ok I see the node version is set up inside the pipeline so it's ok :)

Reply:
yes I checked the same page :)